notes-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java

```diff
@@ -384,6 +384,7 @@ public Response createNote(@Parameter(description = "note object to be created",
       note.setSyntax(syntaxId);
       note.setName(TitleResolver.getId(note.getTitle(), false));
       note.setUrl("");
+      note.setAuthor(currentUser);
```

Comment:
Should be centralized in Service layer, since the identity ACL is passed to Service (remember, in REST, no business logic should be done, which unfortunately not the case of a lot of stuff in this REST endpoint, but at least we need to avoid making it worse)

```suggestion
```